### PR TITLE
Add "User interface" as option in bug report form

### PR DESCRIPTION
**Pull request by Broxzier**

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I’m not sure that the average user will understand this. Heck, I’m not even why you’re adding it in the first place.

**Comment by Broxzier** [comment]
@Gymnasiast I'm sure the average user that reports issues understands this. It could be written as "There is a problem with the user interface" or something to make it easier to understand?

I'm adding this because there are quite some reports being made that have to do with things that are broken in the user interface. Invalid text, weird artefacts, missing or misplaced widgets. Just to link a few to give you an impression:

 - https://github.com/OpenRCT2/OpenRCT2/issues/15439
 - https://github.com/OpenRCT2/OpenRCT2/issues/15844
 - https://github.com/OpenRCT2/OpenRCT2/issues/15257
 - https://github.com/OpenRCT2/OpenRCT2/issues/14310
 - https://github.com/OpenRCT2/OpenRCT2/issues/14135
 - https://github.com/OpenRCT2/OpenRCT2/issues/13758

**Comment by ZehMatt** [comment]
I think it makes sense since we also try to keep UI away from core logic.

**Comment by Gymnasiast** [comment]
I don’t see what it would tell us. It should be obvious from the description anyway, unlike "also present in vanilla" and "multiplayer-specific". We have dozens of labels we can apply to bug reports to class them, what makes this one so special that the user has to do it? It’s just another thing they will fill in incorrectly.

**Comment by Broxzier** [comment]
What's "special" about this, is that it is an area that is often getting reported, and it's exactly an answer to what the question is about ("Area(s) with issue?"). Of course, reports should be obvious, but as you by no doubt are aware, that is not always the case, so this field may even help clarify what a report is about exactly.

I'm hoping that at some point, GitHub implements a feature that allows automatic labelling depending on the given answers, but unfortunately that is not yet supported.

**Comment by Gymnasiast** [comment]
How will the user know whether pressing a button and the game not doing what they want is caused by the interface, or by (for example) the game actions? Hard for the user to determine, but easy for us.

The field in question is for things that are easy for the user to determine yet hard for us. Whether a problem is present on vanilla or only happening on multiplayer is much easier to determine for the user, but also a detail that they would probably leave out if not asked.

So basically, I think what you want to do is not going to provide us with anything useful, is a burden on the user, and is most likely only going to confuse everyone (bad information is worse than no information at all).
